src/bikes/bike.model.ts

```diff
@@ -1,11 +1,14 @@
 import * as mongoose from 'mongoose';
 import { BikeType } from './bike.type';
 
-export const BikeSchema = new mongoose.Schema({
-  type: { type: BikeType, required: true },
-  isElectric: { type: Boolean, required: true },
-  frameSize: { type: Number, required: true },
-});
+export const BikeSchema = new mongoose.Schema(
+  {
+    type: { type: BikeType, required: true },
+    isElectric: { type: Boolean, required: true },
+    frameSize: { type: Number, required: true },
+  },
+  { optimisticConcurrency: true },
```

Comment:
I'm not sure if we need to watch if document has changed as they won't change very often.

Reply:
That's right, but I think it won't hurt anything.